JavaScript/Topics/3.Web_Browser/HTML/1.External_Js.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <script src = "../JS/script.js"></script>
```

Comment:
Insert the script tag just before the closing of the body tag, not saying its incorrect but thik of the js file contains that code which will alter the html because it is present in the head tag, it will load first before body and unable to access DOM elements and it will throw an error, that's why move it to before closing of the body tag.

Also it will be a plus point if you write about adding js file in head tag and at the end of the body tag, write about differences